src/vocabulary/model.rs

```diff
@@ -0,0 +1,284 @@
+                )?;
+            } else if let Some(parent_id) = term.parent_id {
+                graph.insert(
+                    &Iri::new(format!("{}{}", base, term.id).as_str())?,
+                    &a_token,
+                    &skos.get("Concept")?,
+                )?;
+
+                graph.insert(
+                    &Iri::new(format!("{}{}", base, term.id).as_str())?,
+                    &skos.get("prefLabel")?,
+                    &Literal::<String>::new_lang_unchecked(term.name, lang.clone()),
+                )?;
+
+                if parent_id == root_id {
+                    graph.insert(
+                        &Iri::new(format!("{}{}", base, root_id).as_str())?,
+                        &skos.get("hasTopConcept")?,
+                        &Iri::new(format!("{}{}", base, term.id).as_str())?,
+                    )?;
+                    graph.insert(
+                        &Iri::new(format!("{}{}", base, term.id).as_str())?,
+                        &skos.get("topConceptOf")?,
+                        &Iri::new(format!("{}{}", base, root_id).as_str())?,
+                    )?;
+                } else {
+                    graph.insert(
+                        &Iri::new(format!("{}{}", base, parent_id).as_str())?,
+                        &skos.get("narrower")?,
+                        &Iri::new(format!("{}{}", base, term.id).as_str())?,
+                    )?;
+                    graph.insert(
+                        &Iri::new(format!("{}{}", base, term.id).as_str())?,
+                        &skos.get("broader")?,
+                        &Iri::new(format!("{}{}", base, parent_id).as_str())?,
+                    )?;
+                    graph.insert(
+                        &Iri::new(format!("{}{}", base, term.id).as_str())?,
+                        &skos.get("inScheme")?,
+                        &Iri::new(format!("{}{}", base, 3).as_str())?,
+                    )?;
+                }
+            }
+        }
+
+        let prefixes = [
+            (Prefix::new_unchecked("dct"), Iri::new_unchecked(DCT)),
+            (Prefix::new_unchecked("skos"), Iri::new_unchecked(SKOS)),
+        ];
+        let config = TurtleConfig::new()
+            .with_pretty(true)
+            .with_prefix_map(&prefixes[..]);
+
+        let mut output = vec![];
+        let mut serializer =
+            sophia::serializer::turtle::TurtleSerializer::new_with_config(&mut output, config);
+        serializer.serialize_graph(&graph)?;
+
+        let output = format!(
+            "BASE <{}>\n{}",
+            base,
+            String::from_utf8(output)?.replace(&base, "")
+        );
+        Ok(output)
+    }
+}
+
+#[derive(Clone, Debug, Eq, PartialEq, Deserialize, Serialize)]
+#[serde(rename_all = "kebab-case")]
+pub enum TaxonomyType {
+    Root, // Level 0
+
+    Blog, // below Root
+
+    ForumCategory, // below Root or ForumCategory
+    Forum,         // below ForumCategory
```

Comment:
Do we want to report them or shall we filter them?

Reply:
(They are filtered out by https://github.com/serlo/serlo.org-database-layer/pull/141/files?diff=split&w=1#diff-90656dae891493653ff080c4b5d9b9ce1fd3fa012e10b95c5eec2c49ab10c7a4R102-R104 right?)

Reply:
As long as we have such entries in the database (which we do), we need to list them here, otherwise https://github.com/serlo/serlo.org-database-layer/pull/141/files?diff=split&w=1#diff-90656dae891493653ff080c4b5d9b9ce1fd3fa012e10b95c5eec2c49ab10c7a4R100 will fail.